interfaces/inc/azure_c_pal/refcount.h

```diff
@@ -26,7 +26,7 @@ will interact with deallocated memory / resources resulting in an undefined beha
 #include "azure_c_pal/gballoc.h"
 
 // Include the platform-specific file that defines atomic functionality
-#include "refcount_os.h"
+#include "azure_c_pal/interlocked.h"
```

Comment:
yay for switching this over! Can we move this header (refcount.h) to the "common" folder for "things that we need in this repo as dependencies but aren't really a PAL"? I think the "interfaces" folder should just be for PAL interfaces that have different OS implementations

Reply:
Alright, moved it

---
In reply to: [460141738](https://github.com/Azure/azure-c-pal/pull/16#discussion_r460141738) [](ancestors = 460141738)